torchvision/transforms/transforms.py

```diff
@@ -692,6 +692,10 @@ def __init__(self, distortion_scale=0.5, p=0.5, interpolation=InterpolationMode.
 
         self.interpolation = interpolation
         self.distortion_scale = distortion_scale
+
+        if not isinstance(fill, (Sequence, numbers.Number)):
```

Comment:
Maybe, we can keep BC by doing:
```python
if fill is None:
    fill = 0
```
Sorry, I'm a bit hesitating about introducing new behaviour.

Reply:
Agreed. Maybe throw a warning with it or something?

Reply:
@vfdev-5

Reply:
In some sense it was already done by PIL in `_parse_fill*` method, so let's just reassign it without a warning.